relay-dynamic-config/src/defaults.rs

```diff
@@ -70,6 +70,20 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
 
     let features = &project_config.features;
 
+    // If there are any spans in the system, extract the usage metric for them:
+    let any_spans = features.has(Feature::ExtractSpansFromEvent)
+        || features.has(Feature::StandaloneSpanIngestion)
+        || features.has(Feature::ExtractCommonSpanMetricsFromEvent);
```

Comment:
You'd need to add `Feature::ExtractAddonsSpanMetricsFromEvent` as well so we extract that metric for those spans too.